Diagnostics/HealthChecker/Analyzer/Security/Invoke-AnalyzerSecuritySettings.ps1

```diff
@@ -332,7 +332,8 @@ function Invoke-AnalyzerSecuritySettings {
     $additionalDisplayValue = [string]::Empty
     $smb1Settings = $osInformation.Smb1ServerSettings
 
-    if ($osInformation.BuildInformation.MajorVersion -gt [HealthChecker.OSServerVersion]::Windows2012) {
+    if ($osInformation.BuildInformation.BuildVersion -ge "10.0.0.0" -or
+        $osInformation.BuildInformation.MajorVersion -like "Windows2012*") {
```

Comment:
This would also match for `Windows2012` so we should change it to `-like "Windows2012R*"`:

<img width="632" alt="image" src="https://user-images.githubusercontent.com/40993616/217801305-882fb0ed-a873-489b-abea-e624042a48e3.png">